content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,124 @@
+# plots modify
+
+Modify [plot metrics](/doc/command-reference/plots) by specifying and saving
+plotting options.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t [TEMPLATE]] [-x X] [-y Y]
+                        [--no-csv-header] [--title TITLE] [--xlab XLAB]
+                        [--ylab YLAB] [--unset [UNSET [UNSET ...]]]
+
+positional arguments:
+  target                Plot file to set props to.
+```
+
+## Description
+
+It might be not convinient for users as well as automation systems to specify
+all the visualization parameters (such as `y-axis`, `template`, `title` and
+others) each time when plots are generated. The `dvc plots modify` command sets
+(and unsets) default visualization options for plot files. The options affect
+all the visualization commands: `dvc plots show` and `dvc plots diff`.
+
+Target plot file needs to be a plot output (see `dvc run`) of one of the DVC
+pipeline stages and it needs to be stored in `dvc.yaml file`. The plot
+modification command adds the options to `dvc.yaml file`.
+
+## Options
+
+- `-t [TEMPLATE], --template [TEMPLATE]` - set a
+  [plot template](/doc/command-reference/plots#plot-templates).
+
+- `-x X` - set name of the X axis of the plot from the plot file.
+
+- `-y Y` - set name of the Y axis of the plot from the plot file.
+
+- `--no-csv-header` - lets DVC know that CSV or TSV `targets` do not have a
+  header.
+
+- `--title TITLE` - plot title for visualization.
+
+- `--xlab XLAB` - title of X axis.
+
+- `--ylab YLAB` - title of Y axis.
+
+- `--unset [UNSET [UNSET ...]]` - unset an option or option list.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+The initial plot was showing the last column of CSV file by default which is
+_loss_ metrics while _accuracy_ is expected as Y axis:
+
+```CSV
```

Comment:
````suggestion
```csv
````
But really this language is not supported (see https://prismjs.com/#supported-languages) so these blocks dont have special highlighting.

Reply:
👍  we can even consider using  MD table instead?

Reply:
not a good idea ... let's keep a raw file, to keep it closer to the actual file, the way DVC expects it to be

Reply:
So remove `csv` from md code blocks?

Reply:
I guess let's decide follow up #1390 🙂